### DISCUSSION TITLE
[GENERAL] Arena Spectator NPC

### DISCUSSION THREAD

**Discussion by V3nomAnc3r** [azerothcore/azerothcore-wotlk#3956] [General]
Hey, Can some one update this npc for azerothcore

NPC - [ArenaSpectator](https://paste.ubuntu.com/p/D3xq9FhvKv/)
DIFF - [ArenaSpectator](https://github.com/Flameshot/TrinityCore/commit/f475cc36c9904cafbd28cef588bac1c8b29bf44e#diff-cbbcefba2af7a4b40ec13c3f4be7477cR369)

**Comment by Viste** [comment]
lol in AzerothCore spectator system from sunwellcore you dont need npc just addon to client :)

**Comment by Viste** [comment]
https://github.com/Kittnz/Sunwell/tree/master/extras/addons

**Comment by V3nomAnc3r** [comment]
@Viste I know but NPC is better for players :P
